js/src/tooltip.js

```diff
@@ -190,14 +194,14 @@ const Tooltip = (() => {
 
       if (event) {
         const dataKey = this.constructor.DATA_KEY
-        let context = $(event.currentTarget).data(dataKey)
+        let context = Data.getData(event.delegateTarget, dataKey)
 
         if (!context) {
           context = new this.constructor(
-            event.currentTarget,
+            event.delegateTarget,
             this._getDelegateConfig()
           )
-          $(event.currentTarget).data(dataKey, context)
+          Data.setData(event.delegateTarget, dataKey, context)
```

Comment:
I don't think `setData` handle 3 arguments :thinking:

Reply:
Signature is `setData(instance, key, data)`

Reply:
Oops, my bad :+1: